docs/UnInstall_IML.md

```diff
@@ -0,0 +1,184 @@
+[**Table of Contents**](index.md)
+
+# Uninstalling IML
+
+![clustre](md_Graphics/uninstall_sm.jpg)
+
+# **Step 1**: Uninstall the IML Manager
+
+### Stop the manager and httpd 
+
+```
+chroma-config stop
+```
+
+### Remove the Manager
+```
+chkconfig --del chroma-supervisor
+yum clean all --enablerepo=*
+yum remove chroma* fence-agents*
+rm -rf /var/lib/chroma/*
+rm -rf /usr/share/chroma-manager/
+```
+
+### Optionally, remove HTTPD
+```
+yum remove httpd
+For IEEL 3.0, 
+yum remove nginx
+```
+
+### Reset NTP
+```
+mv -f /etc/ntp.conf.pre-chroma /etc/ntp.conf
+```
+
+### Clean up these RPM included files/dirs
+```
+rm -rf /usr/lib/python2.6/site-packages/chroma*
```

Comment:
2.6 -> 2.7

Reply:
Why is this even necessary?  It's a bug if it really does need to be done.  Please file a ticket detailing what needs to be removed from here once the chroma packages are removed.